### DISCUSSION TITLE
Projects (Beta) for Mobile

### DISCUSSION THREAD

**Discussion by MaxM-FF** [community/community#15110] [Mobile] [answered]
Please enable Projects (Beta) on mobile with it's very own category link (akin to Issues and Discussions) along with the ability to edit and move cards in table and board modes.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the note @MaxM-TFF — we're working on bringing Projects support to Mobile right now, and hope to have the first steps for this ship soon.

**Reply by MaxM-FF** [reply]
Amazing to hear that this is currently in progress! Thanks for the update and reply @brianlovin!

**Reply by candyho** [reply]
Hey👋just want to inform you we've just shipped [view and edit projects(beta) custom fields]( https://github.blog/changelog/2022-05-09-view-and-edit-projectsbeta-custom-fields-on-github-mobile/) to both iOS and Android last week! You can now edit custom text, number, date, and single-select fields for issues and pull requests that belong to the new projects experience from within the triage sheet. Try it out and let us know what you think!😄
